website/content/docs/nia/cli/task.mdx

```diff
@@ -24,6 +24,7 @@ In addition to [general options](/docs/nia/cli/cli-overview#general-options) thi
 | Name         | Required | Type   | Description                                                                                                         |
 | ------------ | -------- | ------ | ------------------------------------------------------------------------------------------------------------------- |
 | `-task-file` | Required | string | The path to the task configuration file. See [configuration information](/docs/nia/configuration#task) for details. |
+| `-auto-approve` &nbsp; | Optional | boolean | Whether to skip the interactive approval of the plan before creating. |
```

Comment:
The `&nbsp;` was a suggested way to keep the columns at the desired size. Without it, the "approve" part of `-auto-approve` goes to a new line like:

```
-auto-
approve
```
Which just doesn't look as nice, imo!

Reply:
agree, having it on one line looks better. thanks for explaining!